packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -38,7 +39,9 @@ export const Breadcrumb = forwardRef<HTMLElement, HTMLAttributes<HTMLElement>>((
       <>
         <StyledBreadcrumbItem>{child}</StyledBreadcrumbItem>
         <StyledCenteredBreadcrumbItem>
-          <StyledChevronIcon />
+          <StyledChevronIcon>
+            <ChevronRightStrokeIcon />
```

Comment:
Our SVGO [config](https://github.com/zendeskgarden/react-components/blob/main/.svgo.config.js) adds `aria-hidden="true"` to svgs.